modules/hosted-control-planes-pause-reconciliation.adoc

```diff
@@ -10,7 +10,7 @@ If you are a cluster instance administrator, you can pause the reconciliation of
 
 .Procedure
 
-. To pause reconciliation for a hosted cluster and hosted control plane, populate the `pausedUntil` field of the `HostedCluster` resource, as shown in the following examples:
+. To pause reconciliation for a hosted cluster and hosted control plane, populate the `pausedUntil` field of the `HostedCluster` resource, as shown in the following examples. In the examples, the value that is specified for the `pausedUntil` field is wrapped into the command that follows it.
```

Comment:
Until I looked at the code block to see what was going on, I didn't know what "wrapped into the command" meant.

I'd recommend rewriting to be clearer what is meant. Maybe something like:

```suggestion
. To pause reconciliation for a hosted cluster and hosted control plane, populate the `pausedUntil` field of the `HostedCluster` resource, as shown in the following examples. In the examples, the value for `pausedUntil` is defined in an environment variable prior to the command.
```

Also side note for another time, the below commands use "kubectl", but commands in our docs should use "oc".

Reply:
@jparrill Would it affect the accuracy of the commands here if I changed `kubectl` to `oc`?